js/utils/ellipsis.js

```diff
@@ -1,17 +1,5 @@
 define(['jquery', 'bootstrap', 'jQuery.dotdotdot'], function($) {
-
     $(function() {
         $('.ellipsis-dot').dotdotdot();
-
-        $('.ellipsis-tooltip').dotdotdot({
```

Comment:
I can't remember if we already clean up the code base from this class but I think this is the case.
Have you checked ?